app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.spec.tsx

```diff
@@ -30,25 +31,34 @@ const DEFAULT_PROPS = {
 };
 
 describe('InPersonLocationStep', () => {
+  const server: SetupServerApi = setupServer();
+  before(() => {
```

Comment:
I think we'd want this to be in a lifecycle hook, since Mocha would otherwise initialize this as soon as the file is loaded (ignoring any `.skip`, `.only`, etc). I'm not exactly sure all that's involved with this call, but we'd want to avoid any potential side-effects or the performance impact of setting up the server.

I think we could probably do something like what you'd previously done below.

```suggestion
  let server: SetupServerApi;

  before(() => {
    server = setupServer();
```